umongo/abstract.py

```diff
@@ -137,14 +132,8 @@ def serialize_default():
             val = self.default() if callable(self.default) else self.default
             return self.serialize('foo', {'foo': val})
 
-        if marshmallow_missing is self._field_unset:
-            self.marshmallow_missing = serialize_default
-        else:
-            self.marshmallow_missing = marshmallow_missing
-        if marshmallow_default is self._field_unset:
-            self.marshmallow_default = serialize_default
-        else:
-            self.marshmallow_default = marshmallow_default
+        self.marshmallow_missing = kwargs.get('marshmallow_missing', serialize_default)
+        self.marshmallow_default = kwargs.get('marshmallow_default', serialize_default)
```

Comment:
I should have popped from kwargs, and before calling `super.__init__`, otherwise, the `marshmallow_...` params leak to the Schema metadata.

I'll fix that in a further PR addressing https://github.com/Scille/umongo/issues/168.

Reply:
Fixed in https://github.com/Scille/umongo/pull/170.